lib/tasks/taxes.rake

```diff
@@ -23,4 +23,35 @@ namespace :taxes do
       customer.applied_taxes.create!(tax:)
     end
   end
+
+  desc 'Create fees_taxes for existing fees'
+  task create_fees_taxes: :environment do
+    sub_query = Fee.where('taxes_rate > 0').left_joins(:fees_taxes)
+      .group('fees.id')
+      .having('COUNT(fees_taxes.id) = 0')
+      .select(:id)
+
+    Fee.where(id: sub_query).each do |fee|
+      organization = fee.organization || fee.subscription&.customer&.organization
+
+      tax = ::Tax.create_with(
```

Comment:
I think all taxes should be created after running `taxes:migrate_from_vate_rate` task

Reply:
@rsempe we will indeed have to run the `taxes:migrate_from_vate_rate` before, but since some taxes applied to fees might not exists anymore at organization or customer level, I guess we still have to create the missing ones. What do you think?

Reply:
Exact 👍